lesson_07/homeworks_07_panchenko.py

```diff
@@ -0,0 +1,137 @@
+# task 1
+""" Задача - надрукувати табличку множення на задане число, але
+лише до максимального значення для добутку - 25.
+Код майже готовий, треба знайти помилки та випраавити\доповнити.
+"""
+def multiplication_table(number):
+    multiplier = 1
+    while multiplier <= 10:  
+        result = number * multiplier
+        if result > 25:  
+            break  
+        print(f"{number} x {multiplier} = {result}")
```

Comment:
```suggestion
        print(f"{number} x {multiplier} = {result}")
```
функція, яка нічого не повертає - полу-інвалід